shaptools/saputils.py

```diff
@@ -0,0 +1,67 @@
+"""
+Module to utilize SAP Technology components
+
+:author: sisingh
+:organization: SUSE LLC
+:contact: sisingh@suse.com
+
+:since: 2020-03-12
+"""
+#TODO: Add support for other SAPCAR functionalties apart from extraction
+
+from __future__ import print_function
```

Comment:
I don't think we need this import, right?

Reply:
Yes, I used it for testing but can remove this